backend/project_404_T5/settings.py

```diff
@@ -127,6 +127,8 @@
 
 USE_TZ = True
 
+# these 2 URL should be changed when depolying
```

Comment:
Can you leave a more detailed comment on what it has to be changed to in the case people don't know?